Source/WebCore/Modules/mediastream/RTCRtpScriptTransformer.cpp

```diff
@@ -190,18 +191,47 @@ void RTCRtpScriptTransformer::enqueueFrame(ScriptExecutionContext& context, Ref<
     m_readableSource->enqueue(value);
 }
 
-void RTCRtpScriptTransformer::generateKeyFrame(Ref<DeferredPromise>&& promise)
+static std::optional<Exception> validateRid(const String& rid)
+{
+    if (rid.isNull())
+        return { };
+
+    if (rid.isEmpty())
+        return Exception { ExceptionCode::NotAllowedError, "rid is empty"_s };
+
+    if (rid.length() > 255)
```

Comment:
Nit: it would be nice to have and use a named constant

Reply:
Done